crates/pop-parachains/src/up/mod.rs

```diff
@@ -216,7 +220,7 @@ impl Zombienet {
 	///   will use the latest available version).
 	/// * `network_config` - The network configuration to be used to launch a network.
 	/// * `cache` - The location used for caching binaries.
-	async fn relay_chain(
+	async fn _relay_chain(
```

Comment:
Why the `_` prefix? This infers the function is not used.

If it's to not conflict with the other `relay_chain`, maybe something like `relay_chain_config` would be more accurate?

Reply:
_ is a valid function name, its just linting will not complain. Will update.

Reply:
Improved in 3b3774a9